testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -110,8 +110,8 @@ def build_mesh(
                      output_name='mesh_triangles.nc', on_sphere=on_sphere)
 
     print('Step 4. Convert from triangles to MPAS mesh')
-    write_netcdf(convert(xarray.open_dataset('mesh_triangles.nc')),
-                 'base_mesh.nc')
+    write_netcdf(convert(xarray.open_dataset('mesh_triangles.nc'),
+                    graphInfoFileName='graph.info'),'base_mesh.nc')
```

Comment:
Perfect!  My only comment here would be to indent another 5 spaces so the formatting follows the PEP8 convention.

Reply:
Just so I'm clear, indent the second line 5 more spaces?  Is this so it is easier to see it is part of the convert function?

Reply:
If you load `e3sm_unified` you can run
```
autopep8 --in-place --aggressive --aggressive foo.py
```
on your python files for automatic pep8 spacing.